scipy/linalg/_solve_toeplitz.py

```diff
@@ -0,0 +1,117 @@
+# Author: Robert T. McGibbon, December 2014
+
+from numpy import zeros
+
+# FIXME: from numpy.linalg import LinAlgError when it's supported by Pythran
+LinAlgError = RuntimeError
```

Comment:
Can it not handle subclassing of Errors?

Reply:
no it cannot.